src/main/java/com/example/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -32,9 +36,29 @@ public List<BookDto> findAll() {
     }
 
     @Override
-    public BookDto getBookById(Long id) {
+    public BookDto getById(Long id) {
         Book bookToFind = bookRepository.findById(id).orElseThrow(
                 () -> new EntityNotFoundException("Can't find book with id: " + id));
         return bookMapper.toDto(bookToFind);
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
```

Comment:
What happens, if a book with such id doesn't exist?

Reply:
It's OK if such book doesn't exist. Just nothing will happen and in most cases (not all) it's expected behaviour